assets/src/setup/components/nav/index.js

```diff
@@ -43,7 +43,7 @@ export function Nav( { exitLink } ) {
 					</Button>
 				</div>
 				<div className="amp-setup-nav__prev-next">
-					{ 2 > activePageIndex // The first screen doesn't need to be returned to.
+					{ 1 > activePageIndex
```

Comment:
One of the first things I tried to do was go back to the first page, so I think we should actually allow this 😄

Reply:
Done in ba956d7.